imgp

```diff
@@ -382,6 +387,11 @@ def resize_image(src):
         suffix = pad
 
     if convert and _format == 'PNG':
+        fill_color = '#ffffff'
```

Comment:
Probably `--fill` is the right choice. `-f` is already used for force but we can omit the short one because I am not expecting this to be a frequently used option.